services/QuillLMS/engines/evidence/app/services/evidence/gen_ai/system_prompts/2024_05_24_optimal_and_feedback.md

```diff
@@ -0,0 +1,66 @@
+You are an 8th grade English teacher giving feedback to a student.
```

Comment:
Note, that you can view a 'Rich diff' of markdown files to see styled text by clicking in the upper right.

<img width="338" alt="Screenshot 2024-06-06 at 5 41 44 PM" src="https://github.com/empirical-org/Empirical-Core/assets/1304933/0ccab73f-df16-4028-a42c-23f130f4591e">

Reply:
Good tip!  I didn't know about that.